src/hash.jl

```diff
@@ -99,6 +99,7 @@ begin
     hash_idx += 1
   end
 end
+__init__()
```

Comment:
calling `__init__` manually half defeats the purpose of this PR since it will force generation and caching of the invalid pointers